crd-docs/config/apigen-crd.template

```diff
@@ -1,91 +1,22 @@
 ---
-title: {{ .Title }} CRD Schema Reference
-linkTitle: {{ .Title }} CRD Schema Reference
+title: {{ .Title }} CR Reference
+linkTitle: {{ .Title }} CR Reference
 description: |
 {{- if .Description }}
 {{ .Description | indent 2 }}
 {{- else }}
-  Custom resource definition (CRD) schema reference page for the {{ .Title }} CR (kind: {{ .NamePlural }}.{{ .Group }}).
+  Reference page for the {{ .Title }} CR.
   The Kiali Operator will watch for resources of this type and install Kiali according to those resources' configurations.
 {{- end }}
 weight: {{ .Weight }}
-crd:
-  name_camelcase: {{ .Title }}
-  name_plural: {{ .NamePlural }}
-  name_singular: {{ .NameSingular }}
-  group: {{ .Group }}
-  technical_name: {{ .NamePlural }}.{{ .Group }}
-  scope: {{ .Scope }}
-  source_repository: {{ .SourceRepository }}
-  source_repository_ref: {{ .SourceRepositoryRef }}
-  versions:
-{{- range .Versions }}
-    - {{ . -}}
-{{- end }}
-  topics:
-{{- with .Metadata.Topics }}
-{{- range . }}
-    - {{ . -}}
-{{- end }}
-{{- end }}
-{{- with .Metadata.Providers }}
-  providers:
-{{- range . }}
-    - {{ . -}}
-{{- end }}
-{{- end }}
-{{- with .Metadata.Deprecation }}
-  deprecation:
-{{- with .Info }}
-    info: {{ . }}
-{{- end }}
-{{- with .ReplacedBy }}
-    replaced_by:
-      full_name: {{ .FullName }}
-      short_name: {{ .ShortName }}
-{{- end }}
-{{- end }}
-layout: crd
-owner:
-{{- range .Metadata.Owners }}
-  - {{ . -}}
-{{- end }}
-aliases:
-  - /reference/cp-k8s-api/{{ .NamePlural }}.{{ .Group }}/
 technical_name: {{ .NamePlural }}.{{ .Group }}
 source_repository: {{ .SourceRepository }}
 source_repository_ref: {{ .SourceRepositoryRef }}
 ---
 
-# {{ .Title }}
-
-{{ with .Description }}
-<p class="crd-description">{{ . }}</p>
-{{ end -}}
-
-<dl class="crd-meta">
-<dt class="fullname">Full name:</dt>
-<dd class="fullname">{{ .NamePlural }}.{{ .Group }}</dd>
-<dt class="groupname">Group:</dt>
-<dd class="groupname">{{ .Group }}</dd>
-<dt class="singularname">Singular name:</dt>
-<dd class="singularname">{{ .NameSingular }}</dd>
-<dt class="pluralname">Plural name:</dt>
-<dd class="pluralname">{{ .NamePlural }}</dd>
-<dt class="scope">Scope:</dt>
-<dd class="scope">{{ .Scope }}</dd>
-<dt class="versions">Versions:</dt>
-<dd class="versions">
-{{- range .Versions -}}
-<a class="version" href="#{{.}}" title="Show schema for version {{.}}">{{.}}</a>
-{{- end -}}
-</dd>
-</dl>
-
 {{ if .VersionSchemas }}
 {{ range $versionName, $versionSchema := .VersionSchemas }}
 <div class="crd-schema-version">
-<h2 id="{{$versionName}}">Version {{$versionName}}</h2>
 
 {{with .ExampleCR}}
 <h3 id="crd-example-{{$versionName}}">Example CR (all values shown here are the defaults unless otherwise noted)</h3>
```

Comment:
Can you move the `(all values shown here are the defaults unless otherwise noted)` to just be regular text below the heading?

Reply:
change made - now looks like:
![image](https://user-images.githubusercontent.com/2029470/152002087-e337d1a2-53d9-4394-bddd-1ba03ef330e4.png)